### ISSUE TITLE
Delete records from tables with primary keys (postgresql)

### ISSUE THREAD

**Issue by personalsoft-fabianobonin** [dbeaver/dbeaver#14273]
Given the table below, when I open it from "Database Navigator", select "Data" page, select a row and delete it, when I click on "Save", DBeaver shows this dialog:

"There is no physical unique key defined.
some more text..."
[Use all columns]   [Custom Unique Key] [Cancel]

As this table have a primary key defined, DBeaver should use it instead of showing this dialog.

```sql
CREATE TABLE TEST (
	ID INTEGER NOT NULL,
	COLUMN1 VARCHAR NOT NULL,
	CONSTRAINT TEST_PK PRIMARY KEY (ID)
);

INSERT INTO TEST VALUES (1, 'TEST');

```

**Comment by LonwoLonwo** [comment]
Hello @personalsoft-fabianobonin 
I can't reproduce it. DBeaver can detect the existence of the primary key in the table.

Please tell me more about the DBeaver version you are using. And PostgreSQL version.
Steps to reproduce will be very helpful also.

**Comment by personalsoft-fabianobonin** [comment]
Hi!

DBeaver Community 21.2.3.202110151816
PostgreSQL 12.1, compiled by Visual C++ build 1914, 64-bit
PostgreSQL Driver 42.2.20

I created a new workspace, created a new connection to the same database, repeated the test, and it worked as expected.

I switched to my old workspace, and the problem is still there, so it must be something related to older workspace settings.
I tried to "Reset workspace state", to "Reset UI Settings", and to create a new connection, without success.

I am going to create a new workspace, so I am tagging this issue as closed.

Thanks for your support.

Best regards,

Fabiano
